### ISSUE TITLE
[BUG] openapi-Normalizer usage in config.yaml file doesn't seem to do anything

### ISSUE THREAD

**Issue by roishacham** [OpenAPITools/openapi-generator#18422]
##### Description

I have an API specification that uses tags for the different endpoints. I use 8 tags total. Main reason i used those tags is for better organization within the swagger UI. 
When i generate a python client for this API, it generates 8 different API's, one for each tag, instead of just one like it previously did (before i added the tag).

I've seen [this issue](https://github.com/OpenAPITools/openapi-generator/issues/8121) that seems to be similar, and the solution provided was using the SET_TAGS_FOR_ALL_OPERATIONS tag of openapi-normalizer (which does seem appropriate for my case). So i added this specification to my python_client_config.yaml:

```
openapiNormalizer:
  SET_TAGS_FOR_ALL_OPERATIONS: some_shared_tag
``` 

And i auto-generate my python client, passing this config file as argument.

But, it does nothing, and i still get 8 API's auto-generated.

##### openapi-generator version

7.4.0

**Comment by wing328** [comment]
that's fixed in v7.5.0. please give it a try

**Comment by gero-vachkov** [comment]
@wing328 I'm using the `openapi-generator-maven-plugin` and it seems that `7.5.0` was not published in maven

**Comment by wing328** [comment]
right, i will take a look tomorrow.

please use the CLI as a workaround for the time being

**Comment by wing328** [comment]
looks like it's published correctly: https://repo1.maven.org/maven2/org/openapitools/openapi-generator-maven-plugin/7.5.0/openapi-generator-maven-plugin-7.5.0.jar

**Comment by wing328** [comment]
it's in search.maven.org as well: https://search.maven.org/artifact/org.openapitools/openapi-generator-maven-plugin/7.5.0/maven-plugin

**Comment by gero-vachkov** [comment]
@wing328 you're right. That's odd. Before I wrote the comment the dependency couldn't be resolved, and I also checked https://mvnrepository.com/artifact/org.openapitools/openapi-generator-maven-plugin (the version is still not listed there). I just tested again and was able to download the dependency.
Sorry for bothering you 🙈

**Comment by wing328** [comment]
No problem :)

**Comment by roishacham** [comment]
> that's fixed in v7.5.0. please give it a try

That does fix it! thanks.

The reason we use v7.4.0 in the first place is that when generating C#, we get many build errors, especially:

```
error CS1525: Invalid expression term ';' 
``` 

and

```
error CS1001: Identifier expected 
``` 

All occurring during in our models generation. I was trying to find an opened issue about this, but couldn't (find a few maybe related issues about v7.4.0, but weirdly that's the reason where the C# generation does work for us).

For 2 follow up questions:
1. Are you aware of this issue? if not i can open another issue.
2. Would using v7.5.0 for python client can cause problems if we use v7.4.0 for csharp server?

**Comment by wing328** [comment]
please open another issue with details for tracking.
